src/sentry/monitors/models.py

```diff
@@ -203,7 +204,7 @@ class Monitor(Model):
     API endpoints.
     """
 
-    slug = models.SlugField()
+    slug = SentryOrgSlugField()
```

Comment:
I think this should be `SentrySlugField`, monitors use the standard regex

Reply:
But the comments imply that this is related to an org slug. Are the comments out dated here?

Reply:
```
"""
Organization unique slug of the monitor. Used to identify the monitor in
check-in payloads. The slug can be changed.
"""
```

Reply:
We can also check redash to make sure what the current state of the data is, and which regex is falls in line with

Reply:
I think that comment means the monitor slug has to be unique to the org, not that it is an org slug.
I tested it in [here](https://sentry-test.sentry.io/crons/-asdf/?statsPeriod=14d), and I think it does follow standard b/c it's allowed to start with `-`

Reply:
Got it, will change it to the regular value then